qkcdb/database.go

```diff
@@ -31,25 +31,24 @@ func NewRDBDatabase(file string, clean, isReadOnly bool) (*RDBDatabase, error) {
 	if err := os.MkdirAll(file, 0700); err != nil {
 		return nil, err
 	}
+
 	opts := gorocksdb.NewDefaultOptions()
 	if clean {
 		if err := gorocksdb.DestroyDb(file, opts); err != nil {
 			return nil, err
 		}
 	}
 	// ubuntu 16.04 max files descriptors 524288
-	opts.SetMaxFileOpeningThreads(handles)
+	//opts.SetMaxFileOpeningThreads(handles)
 	// 128 MiB
-	opts.SetMaxTotalWalSize(uint64(cache * 1024 * 1024))
 	// sets the maximum number of write buffers that are built up in memory.
 	opts.SetMaxWriteBufferNumber(3)
 	// sets the target file size for compaction.
-	opts.SetTargetFileSizeBase(6710886)
+	opts.SetTargetFileSizeBase(67108864)
 	opts.SetCreateIfMissing(true)
-	opts.IncreaseParallelism(3)
-	opts.SetMaxBackgroundFlushes(1)
+	opts.SetMaxOpenFiles(100000)
```

Comment:
why set it to 100000?

Reply:
https://github.com/QuarkChain/pyquarkchain/blob/947fc4c10d0c60982b724e3e974738e2ad2aab05/quarkchain/db.py#L68   same as pyquarkchain